syft/formats/common/spdxhelpers/to_syft_model.go

```diff
@@ -49,6 +51,125 @@ func ToSyftModel(doc *spdx.Document) (*sbom.SBOM, error) {
 	return s, nil
 }
 
+func isDirectory(name string) bool {
+	if name == "." || name == ".." || strings.HasSuffix(name, "/") || !strings.Contains(path.Base(name), ".") {
```

Comment:
the check `!strings.Contains(path.Base(name), ".")` doesn't help in all cases to verify if a path is a file or a directory (e.g. `status.d` is a common convention for a directory).

Reply:
Yes, I'm aware of this. It's a best-guess in some case where we have an SBOM that probably wasn't generated by syft. It's a best guess... happy for suggestions here...